client/src/components/TreeDropdownField/TreeDropdownField.js

```diff
@@ -843,6 +748,14 @@ function mapDispatchToProps(dispatch) {
 
 const ConnectedTreeDropdownField = connect(mapStateToProps, mapDispatchToProps)(TreeDropdownField);
 
-export { TreeDropdownField, ConnectedTreeDropdownField, MULTI_EMPTY_VALUE, SINGLE_EMPTY_VALUE };
+export {
+  TreeDropdownField,
+  ConnectedTreeDropdownField,
+  MULTI_EMPTY_VALUE,
+  SINGLE_EMPTY_VALUE,
+  findTreePath,
+  findTreeByID,
+  findTreeByPath,
+};
```

Comment:
I think it would be worth exposing `findTreePath` and `findTreeByID` as customisable props as well, what do you reckon?